modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -828,7 +828,16 @@ private void _extractIndexableAttributes(
 			}
 			catch (Exception exception) {
 				if (_log.isWarnEnabled()) {
-					_log.warn(exception);
+					_log.warn(
+						StringBundler.concat(
+							"Field removal precludes its indexing. The field ",
+							ddmFormField.getName(), " on the form ",
+							ddmStructure.getName(locale),
+							" was removed. Enable DEBUG logs for stacktrace."));
```

Comment:
Can you change the message to:
`"Unable to index field ", ddmFormField.getName(), " because it was deleted from the DDM structure ID",  "ddmStructure.getStructureId()"`